File: eBordo.Api/Database/TrenerStatistika.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace eBordo.Api.Database
{
    public class TrenerStatistika
    {
        public int trenerStatistikaID { get; set; }
        public int brojUtakmica { get; set; }
        public int brojPobjeda { get; set; }
        public int brojNerjesenih { get; set; }
        public int brojPoraza { get; set; }
        public Trener trener { get; set; }
        public int? trenerID { get; set; }
    }
}

File: ebordo_mobile/lib/models/trener-statistika.dart
class TrenerStatistika {
  final int trenerStatistikaID;
  final int brojUtakmica;
  final int brojPobjeda;
  final int brojNerjesenih;
  final int brojPoraza;

  TrenerStatistika(
      {required this.trenerStatistikaID,
      required this.brojUtakmica,
      required this.brojPobjeda,
      required this.brojNerjesenih,
      required this.brojPoraza});

  factory TrenerStatistika.fromJson(Map<String, dynamic> json) {
    return TrenerStatistika(
      trenerStatistikaID: json["trenerStatistikaID"],
      brojUtakmica: json["brojUtakmica"],
      brojPobjeda: json["brojPobjeda"],
      brojNerjesenih: json["brojNerjesenih"],
      brojPoraza: json["brojPoraza"],
    );
  }

  Map<String, dynamic> toJson() => {
        "trenerStatistikaID": trenerStatistikaID,
        "brojUtakmica": brojUtakmica,
        "brojPobjeda": brojPobjeda,
        "brojNerjesenih": brojNerjesenih,
        "brojPoraza": brojPoraza,
      };
}
